axon-server-connector/src/main/java/org/axonframework/axonserver/connector/AxonServerConfiguration.java

```diff
@@ -358,6 +362,14 @@ public boolean isEnabled() {
         return enabled;
     }
 
+    public boolean getLocalSegmentShortCut() {
```

Comment:
I know this is nitty-gritty, but then so is the compiler :)
The correct spelling is shortcut (single word), so the method (and similarly the other method and field) should be called `getLocalSegmentShortcut`.
Given that this shortcut is only usable for queries, I would also include that in the name.
I suggest: `isShortcutQueriesToLocalHandlers()`